game-server/data/handlers/ai/instance/empyreanCrucible/VanktristAI.java

```diff
@@ -0,0 +1,144 @@
+package ai.instance.empyreanCrucible;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.aionemu.commons.utils.Rnd;
+import com.aionemu.gameserver.ai.AIName;
+import com.aionemu.gameserver.ai.HpPhases;
+import com.aionemu.gameserver.model.gameobjects.Creature;
+import com.aionemu.gameserver.model.gameobjects.Npc;
+import com.aionemu.gameserver.model.gameobjects.player.Player;
+import com.aionemu.gameserver.utils.PacketSendUtility;
+import com.aionemu.gameserver.utils.ThreadPoolManager;
+import com.aionemu.gameserver.world.WorldPosition;
+
+import ai.AggressiveNoLootNpcAI;
+
+/**
+ * @author w4terbomb
+ */
+@AIName("vanktrist")
+public class VanktristAI extends AggressiveNoLootNpcAI implements HpPhases.PhaseHandler {
+
+	private final HpPhases hpPhases = new HpPhases(75, 50);
+
+	private Future<?> phaseTask;
+	private final AtomicBoolean isAggred = new AtomicBoolean(false);
+	private final AtomicBoolean isStartedEvent = new AtomicBoolean(false);
+
+	private VanktristAI(Npc owner) {
+		super(owner);
+	}
+
+	@Override
+	protected void handleAttack(Creature creature) {
+		super.handleAttack(creature);
+		if (isAggred.compareAndSet(false, true)) {
+			PacketSendUtility.broadcastMessage(getOwner(), 0);
+		}
+		checkPercentage(getLifeStats().getHpPercentage());
```

Comment:
Everything should be handled via `handleHpPhase`, so `checkPercentage` as well as `isStartedEvent` can be removed. Refer to https://github.com/beyond-aion/aion-server/pull/5#discussion_r1695500455 for the correct usage.